scripts/setup.sh

```diff
@@ -36,11 +41,46 @@ OKTA_REGISTRY=${ARTIFACTORY_URL}/api/npm/npm-okta-master
 echo "Replacing $YARN_REGISTRY with $OKTA_REGISTRY within yarn.lock files..."
 sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" yarn.lock
 
-if ! yarn install ; then
+# Install dependencies but do not build
+if ! yarn install --frozen-lockfile --ignore-scripts; then
   echo "yarn install failed! Exiting..."
   exit ${FAILED_SETUP}
 fi
 
 # Revert the original change(s)
 echo "Replacing $OKTA_REGISTRY with $YARN_REGISTRY within yarn.lock files..."
 sed -i "s#${OKTA_REGISTRY}#${YARN_REGISTRY}#" yarn.lock
+
+# Install a specific version of auth-js, used by downstream artifact builds
+if [ ! -z "$AUTHJS_VERSION" ]; then
+  echo "Installing AUTHJS_VERSION: ${AUTHJS_VERSION}"
+  npm config set strict-ssl false
+
+  AUTHJS_URI=https://artifacts.aue1d.saasure.com/artifactory/npm-topic/@okta/okta-auth-js/-/@okta/okta-auth-js-${AUTHJS_VERSION}.tgz
+  if ! yarn add -DW --ignore-scripts ${AUTHJS_URI}; then
+    echo "AUTHJS_VERSION could not be installed: ${AUTHJS_VERSION}"
+    exit ${FAILED_SETUP}
+  fi
+
+  # NOTE: no additional auth-js versions are included in this repo. If this changes ensure the downstream auth-js is used in all test envs/apps
+  # Example here: https://github.com/okta/okta-react/blob/58883b02f95aeb21befe61ae60e11e4591c792d6/scripts/setup.sh#L69
+
+  npm config set strict-ssl true
+  echo "AUTHJS_VERSION installed: ${AUTHJS_VERSION}"
```

Comment:
Do we have a way of verifying that the requested version of AuthJS is actually installed, and failing otherwise? Worried about false positives

Reply:
Yup, https://github.com/okta/okta-vue/pull/107/files#diff-ce435f38623d032d25670f2c63a430dfd1a02db661d10c1ef031cbef2f0f0ef0R73

Reply:
I see that looks for additional authJS installs, but are we verifying that the single version that is installed matches the version we requested?

Reply:
Added some checks https://github.com/okta/okta-vue/pull/107/files#diff-ce435f38623d032d25670f2c63a430dfd1a02db661d10c1ef031cbef2f0f0ef0R84